build-wheelhouse/action.yml

```diff
@@ -89,7 +89,14 @@ runs:
     - name: "Verify library is properly installed and get its version number"
       shell: bash
       run: |
-        echo "library_version=$(python -c "from ${{ inputs.library-namespace }} import __version__; print(__version__)")" >> $GITHUB_ENV
+        library_version=$(python -c "from ${{ inputs.library-namespace }} import __version__; print(__version__)" || exit 1 )
+        if [ -z "$library_version" ]; then
+            echo "Problem getting the library version"
+            exit 1;
+        else
+            echo "The library version is: $library_version";
```

Comment:
I think this `echo` is going to be helpful, this should help address https://github.com/ansys/actions/issues/269 as well

Reply:
Yep, agreed... if we get a multiline result, it should print it out as well. This will close #269